src/ui/molecules/stepper/selector/stepper.selector.ts

```diff
@@ -0,0 +1,76 @@
+import type { StepperState } from 'hook/useStepper'
+import { OrderedMap } from 'immutable'
+import type { DeepReadonly } from 'superTypes'
+import type { Step, StepId, StepIndex, StepStatus } from 'ui/index'
+
+/**
+ * Get the status of the steps.
+ * @param steps The step array.
+ * @returns The map of the status of the step.
+ */
+export const getStatusFromSteps = (steps: DeepReadonly<Step[]>): OrderedMap<StepId, StepStatus> =>
```

Comment:
This one in unnecessary because we'll never have to provide an `immutable` object as a `useStepper` hook parameter.

Furthermore, be careful, this function is not a selector, but rather an utility's one. A selector takes the state (or a piece of the state) as parameter and computes it to return a view model.